docs/src/user/manual/map.md

```diff
@@ -7,9 +7,63 @@ The user is provided with a visual map to interact with to understand the extent
 
 These are some of the components found on the map tab:
 
-1. Search filters: This provides a casual graphical interface to explore individual metadata records to discover items of interest in terms of content, time and location.This is where users can search for metadata by keywords and date.
-2. Layer: This is the individual records that have been made public. Users can select a layer to see the thumbnail of the metadata record.
-3. Zoom functionality: This allows users to zoom in and out on the map.
-4. Base map options: This allows users to choose a specific base map.
+1. **Right Arrow:** Users can view available metadata records by clicking on the `Right Arrow`. When the user clicks on this arrow, a side panel opens with the metadata record list and the search filter.
 
-![Map](img/map-2.png)
+    ![Map](img/map-2.png)
+
+    1. **Search filters:** This provides a casual graphical interface to explore individual metadata records to discover items of interest in terms of content, time and location. This is where users can search for metadata by keywords and date.
+
+        ![searched by keyword](./img/map-3.png)
+
+    2. **Date:** Users can search for metadata records by specifying the start and end dates. Users can enter the dates directly into the fields or use the calendar icon to select the start and end dates.
+
+        ![searched by date](./img/map-8.png)
+
+    3. **Metadata Record:** To visualise the geographical area associated with the `Metadata Record`, users can click on the metadata, triggering its display on the map.
+
+        ![metadata on map](./img/map-9.png)
+
+        1. **Highlighted Area:** The geographic area corresponding to the metadata is highlighted on the map, providing a visual reference for the user.
+
+        2. **Data:** Users can obtain detailed information by clicking anywhere within the highlighted area. This action reveals specific data associated with the metadata, including:
+
+            - DateTime
+            - Description
+            - Keyword
+            - Name
+        
+        This detailed information provides insights into the metadata record, aiding users in understanding its specific attributes and content.
+
+2. **Zoom functionality:** This allows users to zoom in and out on the map.
+ 
+3. **Base map options:** This allows users to choose a specific base map.
+
+![base map options](./img/map-10.png)
```

Comment:
Just to help with formatting we can resize some images

```suggestion
![base map options](./img/map-10.png){: style="height:200px"}
```

Reply:
okay